update.check/src/test/java/org/kaazing/gateway/update/check/UpdateCheckWithRCVersionIT.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.update.check;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.kaazing.gateway.util.InternalSystemProperty.UPDATE_CHECK;
+import static org.kaazing.gateway.util.InternalSystemProperty.UPDATE_CHECK_SERVICE_URL;
+import static org.kaazing.test.util.ITUtil.timeoutRule;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runners.model.Statement;
+import org.kaazing.gateway.server.impl.VersionUtils;
+import org.kaazing.gateway.server.test.GatewayRule;
+import org.kaazing.gateway.server.test.config.GatewayConfiguration;
+import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+import org.kaazing.test.util.MemoryAppender;
+import org.kaazing.test.util.MethodExecutionTrace;
+
+public class UpdateCheckWithRCVersionIT {
+
+    public GatewayRule gateway = new GatewayRule() {
+        {
+            GatewayConfiguration configuration = createGatewayConfiguration();
+            VersionUtils.reset("Gateway", "Kaazing WebSocket Gateway", "5.6.1-RC001", "");
```

Comment:
Please add a comment showing that the whole loading of the version is overridden by the call to reset.